README.md

```diff
@@ -49,17 +57,21 @@ sets/clears all bits at once whenever it touches a packet. Such
 software is likely to be broadly incompatible with "bitwise" users of
 the packet mark.
 
-| Mark value | Software                     |
-|------------|------------------------------|
-| Any        | [OpenShift][openshift-ovn]   |
-| 0x00000800 | [Antrea][antrea]             |
-| 0x1337     | [Istio][istio]               |
-| 0x1e7700ce | [AWS AppMesh][aws-appmesh]   |
+| Mark value | Software                     | Source                           |
+|------------|------------------------------|----------------------------------|
+| Any        | [OpenShift][openshift-ovn]   | [Source code][openshift-ovn-src] |
+| 0x00000800 | [Antrea][antrea]             | [Documentation][antrea-src]      |
```

Comment:
For posterity: While the actual rule that *sets* this bit in Antrea only sets the `0x800` bit, the reason I put this in non-bitwise is that when Antrea clears this bit, it clobbers the entire range rather than just clearing this one bit :/